onnx/inliner/inliner.cc

```diff
@@ -445,13 +445,14 @@ using FunctionMap = std::unordered_map<FunctionIdKey, std::pair<const FunctionPr
 
 using NodeList = google::protobuf::RepeatedPtrField<NodeProto>;
 
+void InlineFunctions(GraphProto& graph, const FunctionMap& map, NameGenerator& name_generator, ModelProto* model, int inline_count = 0) ;
```

Comment:
Possible to add docs to the function signature? For example, it may not be immediately clear what inline_count refers to.